tecken/settings.py

```diff
@@ -6,765 +6,787 @@
-    UPLOAD_REATTEMPT_LIMIT_SECONDS = values.IntegerValue(60 * 60 * 12)
-
-    # When you "upload by download", the URL's domain needs to be in this
-    # allow list. This is to double-check that we don't allow downloads from
-    # domains we don't fully trust.
-    ALLOW_UPLOAD_BY_DOWNLOAD_DOMAINS = values.ListValue(
-        [
-            "queue.taskcluster.net",
-            "firefox-ci-tc.services.mozilla.com",
-            "stage.taskcluster.nonprod.cloudops.mozgcp.net",
-        ]
-    )
+if LOCAL_DEV_ENV or TEST_ENV or TOOL_ENV:
+    LOGGING["handlers"]["console"]["formatter"] = "human"
 
-    # A list of file extensions that if a file is NOT one of these extensions
-    # we can immediately return 404 and not bother to process for anything
-    # else.
-    #
-    # It's case sensitive and has to be lower case.  As a way to get marginal
-    # optimization of this, make sure '.sym' is first in the list since it's
-    # the most common.
-    DOWNLOAD_FILE_EXTENSIONS_ALLOWED = values.ListValue(
-        [".sym", ".dl_", ".ex_", ".pd_", ".dbg.gz", ".tar.bz2"]
-    )
 
+# Defaulting to 'localhost' here because that's where the Datadog
+# agent is expected to run in production.
+STATSD_HOST = _config("STATSD_HOST", default="localhost", doc="statsd host.")
+STATSD_PORT = _config("STATSD_PORT", default="8125", parser=int, doc="statsd port.")
+STATSD_NAMESPACE = _config(
+    "STATSD_NAMESPACE", default="", doc="Namespace for statsd keys."
+)
 
-class Localdev(Base):
-    """Configuration to be used during local development and base class
-    for testing"""
-
-    # Override some useful developer settings to be True by default.
-    ENABLE_TOKENS_AUTHENTICATION = values.BooleanValue(True)
-    DEBUG = values.BooleanValue(default=True)
-    DEBUG_PROPAGATE_EXCEPTIONS = values.BooleanValue(default=True)
-
-    # When doing localdev, these defaults will suffice. The minio
-    # one forces you to use/test boto3 and the old public symbols URL
-    # forces you to use/test the symbol downloader based on requests.get().
-    SYMBOL_URLS = values.ListValue(["http://minio:9000/testbucket"])
-
-    # By default, upload all symbols to this when in local dev.
-    UPLOAD_DEFAULT_URL = values.Value("http://minio:9000/testbucket")
-
-    # Note! By default the value for 'UPLOAD_TRY_SYMBOLS_URL' becomes
-    # the value of 'UPLOAD_DEFAULT_URL' but with a '/try' prefix added.
-
-    # Run this much sooner in local development.
-    UPLOAD_REATTEMPT_LIMIT_SECONDS = values.IntegerValue(60)
-
-    @classmethod
-    def post_setup(cls):
-        super().post_setup()
-        # in case we don't find these AWS config variables in the environment
-        # we load them from the .env file
-        for param in ("ACCESS_KEY_ID", "SECRET_ACCESS_KEY", "DEFAULT_REGION"):
-            if param not in os.environ:
-                os.environ[param] = values.Value(
-                    default="", environ_name=param, environ_prefix="AWS"
-                )
```

Comment:
This doesn't make a lot of sense anymore. I think it's loading things from a `.env` file, but we don't really do that explicitly in the local dev and test environments anymore.